plugins/importexport/csv/CSVImportExportPlugin.php

```diff
@@ -92,214 +130,257 @@ public function display($args, $request)
-            foreach ($authors as $authorString) {
-                // Examine the author string. Best case is: Given1 Family1 <email@address.com>, Given2 Family2 <email@address.com>, etc
-                // But default to press email address based on press path if not present.
-                $givenName = $familyName = $emailAddress = null;
-                $authorString = trim($authorString); // whitespace.
-                if (!preg_match('/^(\w+)([\w\s]+)?(<([^>]+)>)?$/', $authorString, $matches)) {
-                    echo __('plugins.importexport.csv.invalidAuthor', ['author' => $authorString]) . "\n";
+            $this->initializeStaticVariables();
+
+            if ($data->bookCoverImage) {
+                $reason = InvalidRowValidations::validateBookCoverImageInRightFormat($data->bookCoverImage);
+                if ($reason) {
+                    CSVFileHandler::processInvalidRows($fieldsList, $reason, $this->invalidRowsFile, $this->failedRowsCount);
                     continue;
                 }
-                $givenName = trim($matches[1]); // Mandatory
-                if (isset($matches[2])) {
-                    $familyName = trim($matches[2]);
+
+                $srcFilePath = "{$basePath}/{$data->bookCoverImage}";
+
+                $reason = InvalidRowValidations::validateBookCoverImageIsReadable($srcFilePath, $data->title);
+                if ($reason) {
+                    CSVFileHandler::processInvalidRows($fieldsList, $reason, $this->invalidRowsFile, $this->failedRowsCount);
+                    continue;
                 }
-                $emailAddress = $matches[4] ?? $contactEmail;
-                $author = Repo::author()->newDataObject();
-                $author->setData('publicationId', $publicationId);
-                $author->setSubmissionId($submissionId);
-                $author->setUserGroupId($authorGroup->getId());
-                $author->setGivenName($givenName, $locale);
-                $author->setFamilyName($familyName, $locale);
-                $author->setEmail($emailAddress);
-                if ($firstAuthor) {
-                    $author->setPrimaryContact(1);
-                    $firstAuthor = false;
+
+                $sanitizedCoverImageName = str_replace([' ', '_', ':'], '-', mb_strtolower($data->bookCoverImage));
+                $sanitizedCoverImageName = PKPString::regexp_replace('/[^a-z0-9\.\-]+/', '', $sanitizedCoverImageName);
+                $sanitizedCoverImageName = basename($sanitizedCoverImageName);
+
+                $coverImageUploadName = uniqid() . '-' . $sanitizedCoverImageName;
+
+                $destFilePath = $this->publicFileManager->getContextFilesPath($pressId) . '/' . $coverImageUploadName;
+                $bookCoverImageSaved =  $this->fileManager->copyFile($srcFilePath, $destFilePath);
+
+                if (!$bookCoverImageSaved) {
+                    $reason = __('plugin.importexport.csv.erroWhileSavingBookCoverImage');
```

Comment:
```suggestion
                    $reason = __('plugin.importexport.csv.errorWhileSavingBookCoverImage');
```
(and in locale file)